docs/source/get_started/index.rst

```diff
@@ -0,0 +1,10 @@
+Get Started
```

Comment:
Getting started, maybe?

Reply:
I thought like it, but Geti used the expression like "GET STARTED" at https://sc-demo.iotg.sclab.intel.com/docs/guide/get-started/supported-tasks.html, which was shared by documentation team.

Reply:
yeah, looks like get started is more popular.
https://textranch.com/277981/getting-started/or/get-started/